src/DocumentSelectionHandler.ts

```diff
@@ -9,6 +9,7 @@ import { HttpCodesEnum } from "./utils/HttpCodesEnum";
 import { DocumentSelectionRequestProcessor } from "./services/DocumentSelectionRequestProcessor";
 import { AppError } from "./utils/AppError";
 import { ssmClient, GetParameterCommand } from "./utils/SSMClient";
```

Comment:
We can get rid of this file then if we're moving the logic over to ./utils/Config